src/user.js

```diff
@@ -7,8 +7,23 @@ class User {
     this.recipesToCook = [];
   }
 
+  validateDataType(attribute, dataType) {
+    return typeof attribute === dataType ? attribute : this.giveDefaultValue(dataType);
+  }
+
+  giveDefaultValue(dataType) {
+    switch (dataType) {
+      case 'string':
+        return 'Invalid value given';
+        break;
+      case 'number':
+        return 0;
+        break;
+    }
+  }
+
```

Comment:
This is really cool, It pretty much just evaluates the typof dataType and changes return value based on if it is a string or number?

Reply:
Great use of the switch statement, I know how to read it but have never used it before, I will give it a try

Reply:
I haven't tried using this switch statement, they look very straight forward and will try to implement some of the moving forward